src/components/App.css

```diff
@@ -0,0 +1,7 @@
+.app {
```

Comment:
Please remove this file and use https://www.styled-components.com/ or a simple object with the css properties you need in the component where you use them.